### PR TITLE
Allow creating project-level scratch orgs.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/a7LEc8Pk/215-as-a-user-i-want-to-spin-up-a-scratch-org-for-a-project-without-creating-a-new-epic-or-task

## Description
_The commit messages say what you did; this should explain why and/or how._
- [x] Description is in Trello card

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- Project Detail sidebar

## Show me
_Provide screenshots/animated gifs/videos if necessary._

Screen Shot 2021-03-29 at 10 22 33 AM

Screen Shot 2021-03-29 at 10 22 37 AM

Screen Shot 2021-03-29 at 10 22 48 AM

### CODE REVIEW

**Comment by jgerigmeyer** [comment]
[As a User, I want to spin up a scratch org for a project without creating a new epic or task.](https://trello.com/c/a7LEc8Pk/215-as-a-user-i-want-to-spin-up-a-scratch-org-for-a-project-without-creating-a-new-epic-or-task)

**Comment by jgerigmeyer** [comment]
@stacyk There isn't style cleanup needed here as far as I can tell, but please review and confirm 👍

**Review by stacyk — APPROVED** [review]
This looks good. I can only set up 1 scratch org for the project, but I think that expected. I edited the markup a bit to help reduce the need for extra `<br />` and such

**Review by duggiemitchell — APPROVED** [review]
LGTM!

File: locales_dev/en/translation.json [hunk 1]
```diff
@@ -323,8 +323,9 @@
...
-  "createScratchOrgHelp": "<0><0>You are creating a Scratch Org for <1>{{type}}</1> “<3>{{name}}</3>”.</0></0><1>Your new org will expire in 30 days.<1></1>You will be able to access your org from this <3>{{type}}</3> page and from the <5>project page</5>.</1><2></2><3>If you would like to create a scratch org for the entire project,<1></1>go to the <3>project page</3> and click “Create Scratch Org” in the sidebar.</3>",
+  "createEpicTaskScratchOrgHelp": "<0>If you would like to create a Scratch Org for the entire Project, <2>go to the Project</2> and click “Create Scratch Org” in the sidebar.</0>",
+  "createProjectScratchOrgHelp": "<0>Visit an Epic or Task to create a Scratch Org from a work in progress.</0>",
+  "createScratchOrgHelp": "<0>You are creating a Scratch Org for <1>{{type}}</1> “<3>{{name}}</3>”.</0><1>Your new org will expire in 30 days.<1></1>You will be able to access your org from this <3>{{type}}</3> page.</1><2><0>You will not be able to retrieve any changes made in this Scratch Org.</0></2>",
```
[the remark this answers is not in the document]
Comment (jgerigmeyer) [reply locales_dev/en/translation.json:328]: They correspond to HTML elements (e.g. `<p><strong>` became `<h3>`, so `<0><0>` became just `<0>`). But really, when you're running the server this gets automatically changed in `locales_dev/en/translation.json` -- we just need to manually copy that change over to the production file: `locales/en/translation.json`.

**Pull request merged**